src/components/Navbar/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Link } from "gatsby";
+import MobileHeader from "./mobile";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import IconButton from "@material-ui/core/IconButton";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import { useTheme } from '@material-ui/core/styles'
+
+import AcountabilityLogo from "../../img/logo.svg";
+
+import HeaderNav from "./HeaderNav";
+import useStyles from "./style";
+
+export default function ButtonAppBar() {
+  const classes = useStyles();
+  const theme = useTheme()
+  return useMediaQuery("(min-width:960px)") ? (
```

Comment:
On my screen, there's about 10px where the alignment just goes crazy - looks like 967px should be the breakpoint

```suggestion
  return useMediaQuery("(min-width:967px)") ? (
```

Reply:
Done